src/cli/NativeMsgManager.cpp

```diff
@@ -0,0 +1,115 @@
+#include "NativeMsgManager.h"
+#include <QCoreApplication>
+#include <QDebug>
```

Comment:
Xtra include?

Reply:
I needed QDebug for `qWarning()` on Windows